scripts/binary_to_array.py

```diff
@@ -48,8 +48,6 @@
 import logging
 import os
 
```

Comment:
What was the problem with the previous state of this script?

Reply:
Most of the context for these changes was in the individual commit messages (e.g. [this one](https://github.com/firebase/firebase-ios-sdk/pull/4984/commits/07d1e5e1a72abb405e11d992b6f73789317ab4da)).

This code previously expected to be run from within a git repository but this assumption is flawed:

  * iOS users can download an archive of the source tree, in which case there's no `.git` folder
  * `firebase-cpp-sdk` actually does this to download its copy of `firebase-ios-sdk`